### ISSUE TITLE
Can't compile version 1.8.1

### ISSUE THREAD

**Issue by FFY00** [f4exb/dsdcc#16]
Here's the CMake command.
```
cmake .. \
    -DCMAKE_BUILD_TYPE=Release \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DLIBSERIALDV_INCLUDE_DIR=/usr/include/serialdv \ 
    -DUSE_MBELIB=ON
```
I double checked the include directories and they are right.

The CMake configuration seems to be fine (appart from the deprecated use of CMP0000).
```
-- The C compiler identification is GNU 8.1.1
-- The CXX compiler identification is GNU 8.1.1
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found LibMbe: /usr/include, /usr/lib/libmbe.so
-- Found libserialdv: /usr/include/serialdv, /usr/lib/libserialdv.so
CMake Warning (dev) in CMakeLists.txt:
  No cmake_minimum_required command is present.  A line of code such as

    cmake_minimum_required(VERSION 3.11)

  should be added at the top of the file.  The version specified may be lower
  if you wish to support older CMake versions for this project.  For more
  information run "cmake --help-policy CMP0000".
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done
-- Generating done
-- Build files have been written to: /home/anubis/git/pkg/dsdcc/src/dsdcc-1.8.1/build
```

And here's the error.
```
/home/anubis/git/pkg/dsdcc/src/dsdcc-1.8.1/dsd_main.cpp: In function ‘int main(int, char**)’:
/home/anubis/git/pkg/dsdcc/src/dsdcc-1.8.1/dsd_main.cpp:270:21: error: ‘serialDevice’ was not declared in this scope
             strncpy(serialDevice, (const char *) optarg, 16);
                     ^~~~~~~~~~~~
/home/anubis/git/pkg/dsdcc/src/dsdcc-1.8.1/dsd_main.cpp:270:21: note: suggested alternative: ‘dvSerialDevice’
             strncpy(serialDevice, (const char *) optarg, 16);
                     ^~~~~~~~~~~~
                     dvSerialDevice
/home/anubis/git/pkg/dsdcc/src/dsdcc-1.8.1/dsd_main.cpp:281:17: error: ‘dvGain_dB’ was not declared in this scope
                 dvGain_dB = (int) (10.0f * log10f(gain));
                 ^~~~~~~~~
/home/anubis/git/pkg/dsdcc/src/dsdcc-1.8.1/dsd_main.cpp:482:150: error: ‘dvGain_dB’ was not declared in this scope
                 dvController.decode(dvAudioSamples, (const unsigned char *) dsdDecoder.getMbeDVFrame1(), (SerialDV::DVRate) dsdDecoder.getMbeRate(), dvGain_dB);
                                                                                                                                                      ^~~~~~~~~
/home/anubis/git/pkg/dsdcc/src/dsdcc-1.8.1/dsd_main.cpp:499:150: error: ‘dvGain_dB’ was not declared in this scope
                 dvController.decode(dvAudioSamples, (const unsigned char *) dsdDecoder.getMbeDVFrame2(), (SerialDV::DVRate) dsdDecoder.getMbeRate(), dvGain_dB);
                                                                                                                                                      ^~~~~~~~~
```

**Comment by f4exb** [comment]
Well too much cleanup... Should be fixed with v1.8.2

**Comment by FFY00** [comment]
Thanks! Works like a charm now :blush:.
